native/packages/react-native-bpk-component-banner-alert/src/AnimateAndFade.js

```diff
@@ -0,0 +1,110 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2017 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {
+  Platform,
+  Animated,
+  ViewPropTypes,
+} from 'react-native';
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import BpkAnimateHeight from 'react-native-bpk-component-animate-height';
+
+const tokens = Platform.select({
```

Comment:
can we use the new and improved way to get the tokens?

 `import { animationDurationSm } from 'bpk-tokens/tokens/base.react.native';`